### PR TITLE
Clean up utils into own submodules

### LINKED ISSUE #1131: Tidy up mslib.utils
The utils module has grown too large.

QT related stuff should be move to "msui.mss_qt". msui.mss_qt might be renamed mss.qt where we're at it.
(most stuff from dropEvent to CheckableComboBox)

The airspace related stuff should get its own submodule mslib.airspace or similarly named.

Config related stuff should go to a config submodule.

Some other stuff might also find a fitting home.

### PR DESCRIPTION
**Pull request by Marilyth**
Fixes #1131 
Cleans up most of utils.py and groups the functions/classes into other submodules
mslib/utils/airdata.py
mslib/utils/config.py
mslib/utils/coordinate.py
mslib/utils/time.py
mslib/msui/mss_qt.py

### CODE REVIEW

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
Much changes :) 


can you please split the tests.py module too. we should have a test_ module for any (new) module in utils.

**Comment by ReimarBauer** [comment]
I'm not happy with the module name "time". But I have also no better idea.

**Comment by Marilyth** [comment]
The tests belonging to a specific utils module have been moved to their own files

**Comment by Marilyth** [comment]
I should not have done that before the GSOC merge, oh god the conflicts.

File: mslib/utils/_tests/test_config.py [hunk 1]
```diff
@@ -1,10 +1,17 @@
 # -*- coding: utf-8 -*-
 """
 
+<<<<<<< HEAD
```
Comment (ReimarBauer) [inline mslib/utils/_tests/test_config.py]: have a look on this merge conflict

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
please see comment in test_config.py

**Review by ReimarBauer — APPROVED** [review]
Thx

**Pull request merged**